multiregion/docs/multiregion.rst

```diff
@@ -1222,10 +1234,52 @@ Stop the Tutorial
 
       ./scripts/stop.sh
 
+.. _multiregion-demo-teardown:
+
+Teardown
+--------
+
+If you ran the tutorial manually, step-by-step, no further action is required.
```

Comment:
The top part of this `Teardown` section duplicates `Stop the Tutorial`.  Can they be collapsed into the former or the latter?

Reply:
Yes I can do that.